orttraining/orttraining/test/python/orttraining_test_orttrainer_bert_toy_onnx.py

```diff
@@ -911,3 +925,57 @@ def testToyBERTModelGradientAccumulationLegacyExperimental(gradient_accumulation
 
     # Check results
     _test_helpers.assert_model_outputs(experimental_losses, legacy_losses, rtol=1e-6)
+
+@pytest.mark.parametrize("params, legacy_optim_map", [
+    # Change the hyper parameters for all parameters
+    ([], legacy_optim_params_a),
+    # Change the hyperparameters for a subset of hardcoded parameters
+    ([{'params':['bert.embeddings.LayerNorm.bias', 'bert.embeddings.LayerNorm.weight'], "alpha": 0.9, "beta": 0.999, "lambda_coef": 0.0, "epsilon": 1e-6}], legacy_optim_params_b),
+    # Change the hyperparameters for a generated set of paramers
+    (optimizer_parameters_old(load_bert_onnx_model()), legacy_optim_params_c)
+])
+def testToyBERTModelLegacyExperimentalCustomOptimParameters(params, legacy_optim_map):
+    total_steps = 10
+
+    # EXPERIMENTAL API
+    model_desc = bert_model_description()
+    model = load_bert_onnx_model()
+
+    #optim_config = optim.LambConfig(params, do_bias_correction=False)
+    optim_config = optim.LambConfig(params, alpha= 0.9, beta= 0.999, lambda_coef= 0.01, epsilon= 1e-6, do_bias_correction=False)
+    opts =  orttrainer.ORTTrainerOptions({
+        'debug' : {
+            'deterministic_compute': True
+        },
+    })
+    
+    torch.manual_seed(1)
```

Comment:
set seed = 1 in the first line of code

Copy the template from previous examples in this file. there is a "#common setup" piece